.github/actions/render/action.yml

```diff
@@ -0,0 +1,7 @@
+name: Render ZIPs and Zcash Protocol Specification
+description: GitHub Action to compile ZIPs and Zcash Protocol Specification LaTeX documents
+author: Deirdre Connolly
+runs:
+  using: docker
+  # Runs `make all` or something like it
+  image: Dockerfile
```

Comment:
This might have to change if you move the file back. Alternatively add a symlink.

Reply:
It's obviously been a while since I did this, but IIRC symlinks and alternate locations didn't work. I can re-investigate I guess.

Reply:
Done.